scripts/ci/create-docker-image.sh

```diff
@@ -4,10 +4,13 @@ set -v
 set -x
 set -e
 
+: ${WITH_SHA1:=false}
```

Comment:
So WITH_SHA1 is treated as an executable?

For treating like a variable, leave empty here, and `[ -n "${WITH_SHA1}" ] && ` later.

Reply:
if not `true` then will set of `false`

Reply:
<pre>
> WITH_SHA1=1
> $WITH_SHA1 && echo its true || echo its false
bash: 1: command not found
its false
</pre>

All other values are variables rather than names of executables.

Reply:
I have fixed this in my last update